RecoJets/JetProducers/python/ak8PFJets_cfi.py

```diff
@@ -26,7 +26,9 @@
     )
 
 from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
-pp_on_AA_2018.toModify(ak8PFJetsCHS,src = "pfNoPileUpJMEHI", inputEtMin = 9999)
+from Configuration.Eras.Modifier_pp_on_PbPb_run3_cff import pp_on_PbPb_run3
+(pp_on_AA_2018 | pp_on_PbPb_run3).toModify(ak8PFJetsCHS, src = "pfEmptyCollection", inputEtMin = 9999)
+(pp_on_AA_2018 | pp_on_PbPb_run3).toModify(ak8PFJetsPuppi, src = "pfEmptyCollection", inputEtMin = 9999)
```

Comment:
are the changes to the cuts really needed if the input is really an empty collection?